<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elevator</class>
 <widget class="QMainWindow" name="Elevator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elevator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="floor1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="doorOpen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Door Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doorClose">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Door Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="obstacle">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Door Obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fire">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overload">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Overload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerout">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="carNum">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="safety">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Safety Scenarios</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
